yarn.lock

```diff
@@ -3764,6 +3764,23 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@metamask/utils@npm:^10.0.0":
```

Comment:
v8 and v9 are still in use by transitive snaps packages, but I guess that's intended to resolve in a follow-up.

Reply:
Yeah, this strictly is meant to unblock usage of the latest utils version in Snaps, since we want a performance improvement is included in the latest version.

Reply:
Could consider holding back update in `@metamask/accounts-controller` to keep the versions consistent withing packages? There's a configuration for allowing that in `yarn.config.cjs`.